### DISCUSSION TITLE
非根目录下斜杠丢失

### DISCUSSION THREAD

**Discussion by Js-Man-H5** [umijs/dumi#1995] [Q&A]
这是nginx配置
![image](https://github.com/umijs/dumi/assets/62057975/5aad9aaf-6ab3-4e15-b707-8967fa17c595)
这是dumi的配置
![image](https://github.com/umijs/dumi/assets/62057975/26f4854a-a9ab-498b-afb5-f8e7ec777e58)
实际请求后
![image](https://github.com/umijs/dumi/assets/62057975/94283e72-0c1b-4f35-8fa1-63266244ca1e)

**Comment by PeachScript** [comment]
dumi 会强制去掉尾部 `/`，没有 `/` 是预期结果
